oauth2client/crypt.py

```diff
@@ -95,7 +97,108 @@ def make_signed_jwt(signer, payload):
     return b'.'.join(segments)
 
 
-def verify_signed_jwt_with_certs(jwt, certs, audience):
+def _verify_signature(message, signature, certs):
```

Comment:
I don't think from a function design point of view that we should be looking for reasons to justify taking more than is needed rather than just paring down the parameter set to only what the function needs? Especially for a private function.

Also AppIdentityError does seem like a rather slim class that includes only a fixed message and none of the actual values that led to its generation - would that benefit from being fleshed out a little?